src/index.js

```diff
@@ -19,6 +19,10 @@ import { LandingPage } from './components/pages/Landing';
 import { ExampleDataViz } from './components/pages/ExampleDataViz';
 import { config } from './utils/oktaConfig';
 import { LoadingComponent } from './components/common';
+import MainHeader from './components/common/Header';
+import MainFooter from './components/common/Footer';
+import instructorHome from './components/pages/InstructorHome';
```

Comment:
Everything looks fine, but we may want to capitalize the first letter of instructorHome since it's a Component and it would be consistent with the rest of the project.

Reply:
Gotcha! I just changed it